introduction.md

```diff
@@ -0,0 +1,2 @@
+Hello, this is Fillipe and that is my first commit to DOJO repository.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.